cropper/src/main/java/com/canhub/cropper/CropWindowHandler.kt

```diff
@@ -193,41 +199,38 @@ class CropWindowHandler {
         targetRadius: Float
     ): CropWindowMoveHandler.Type? {
 
+        if (!isInBounds(x, y, mEdges)) return null
+
         // Note: corner-handles take precedence, then side-handles, then center.
         return when {
-            isInCornerTargetZone(x, y, mEdges.left, mEdges.top, targetRadius) -> {
+            isCloseToHandle(x, y, mEdges.left, mEdges.top, targetRadius) -> {
                 CropWindowMoveHandler.Type.TOP_LEFT
             }
-            isInCornerTargetZone(x, y, mEdges.right, mEdges.top, targetRadius) -> {
+            isCloseToHandle(x, y, mEdges.right, mEdges.top, targetRadius) -> {
                 CropWindowMoveHandler.Type.TOP_RIGHT
             }
-            isInCornerTargetZone(x, y, mEdges.left, mEdges.bottom, targetRadius) -> {
+            isCloseToHandle(x, y, mEdges.left, mEdges.bottom, targetRadius) -> {
                 CropWindowMoveHandler.Type.BOTTOM_LEFT
             }
-            isInCornerTargetZone(x, y, mEdges.right, mEdges.bottom, targetRadius) -> {
+            isCloseToHandle(x, y, mEdges.right, mEdges.bottom, targetRadius) -> {
```

Comment:
My idea was to make `isInCornerTargetZone`'s name more generic, since the handle could actually be anywhere. It only applies to the corner when you give coordinates of a corner handle, and I would like to use it for an edge handle.

Reply:
I've left `isCloseToHandleCorner` as it was, and just shared the code that calculates the distance between two points.